src/App.tsx

```diff
@@ -1,26 +1,145 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID, creatTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { SortField } from './types/SortField';
+import { filterTodos } from './utils/helpers.ts/filterTodos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoList } from './components/Todolist';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState(SortField.All);
+  const [title, setTitle] = useState('');
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  function showErrorMessage(errorMessage: string) {
```

Comment:
Use one style for you functions, as I see in most cases you use arrow functions, but this one is function declaration